plugin/config.json

```diff
@@ -53,6 +53,12 @@
       "description": "Log level - info, warn, error or debug",
       "value": "info",
       "Settable": [ "value"]
+	},
+	{
+      "name": "VMCI_VERSION",
+      "description": "Request VMCI version",
+      "value": "",
```

Comment:
actually - this one is ONLY for debugging and testing, so we can set it to something and make sure the versions mismatch handling is correct

@lipingxue  -- please change name to something more obvious (that it's for test only) and add proper comment in description. Something like `TEST_PROTOCOL_VERSION`